src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -245,31 +251,31 @@ export class TreeGrid<DATA>
         this.collapseNode(node);
     }
 
-    onHighlightingChanged(listener: (node: TreeNode<DATA>, force: boolean, callback: Function) => void) {
+    onHighlightingChanged(listener: () => void) {
         this.highlightingChangeListeners.push(listener);
         return this;
     }
 
-    public getRowByNode(node: TreeNode<DATA>): JQuery {
+    private getRowByNode(node: TreeNode<DATA>): JQuery {
```

Comment:
Codacy found an issue: ['JQuery' is not defined.](https://app.codacy.com/manual/enonic/lib-admin-ui/pullRequest?prid=6401650)